lib/metasploit/framework/login_scanner/zabbix.rb

```diff
@@ -115,14 +115,15 @@ def try_login(credential)
           res = try_credential(credential)
           if res && res.code == 302
             opts = {
-              'uri'     => normalize_uri('profile.php'),
+              #'uri'     => normalize_uri('profile.php'), --> works for versions 3 and 4 but not 5
+              'uri'     => normalize_uri('discoveryconf.php'),
```

Comment:
Probably best to remove this comment in the final version but out of interest have you confirmed that `discoveryconf.php` still exists in all the versions prior to version 3? If not you may be breaking functionality by changing this URL, and it may be better to keep this URL in there and then add a new check in which runs if this current check fails to make sure its not failing cause the target is a newer version of Zabbix.

Reply:
Yes, confirmed that discoveryconf.php exists as of version 1.8.22 up to current

Reply:
Hmm so my thoughts on this are as follows:

Whilst we technically try to avoid supporting seriously outdated OS's that are no longer maintained by the developers, looking at https://www.zabbix.com/rn/rn1.8.22 suggests this OS was released in 2014, or roughly 6 years ago. According to https://www.zabbix.com/life_cycle_and_release_policy the EOL for this software, as in critical updates etc, is 5 years after release, which would mean this version specifically is EOL now with no plans for any more security updates or critical updates.

On the other hand this software is only 6 years old, and its possible people could still be using it in their environment as people do have the tendency to leave legacy software running in their environments.

I'll get back to you on this one, cause on the one hand as a library we should be supporting as many targets as possible, but on the other hand I know there is a limit to how long we can continue supporting old EOL machines.

Reply:
Ok after further review this should be fine, we could in theory go older with certain pages but it seems that some of them don't always have different behavior depending on the versions so this seems like the most reliable option for now.

Reply:
https://github.com/rapid7/metasploit-framework/pull/14297/commits/bd578324941b2ef1f65b452c80c45875f9bf9b68 applies the fixes for the comment part as I mentioned previously.